File: 백준/Bronze/28214. 크림빵/크림빵.py
n, k, p = map(int, input().split())
v = list(map(int, input().split()))
sol = 0

for i in range(n):
    cnt = 0
    for j in range(k):
        if v[i * k + j] == 0:
            cnt += 1
        if cnt < p:
            sol += 1

print(sol)

File: 백준/Bronze/28214. 크림빵/크림빵.cc
#include <bits/stdc++.h>
using namespace std;
#define u ios::sync_with_stdio(false); cin.tie(NULL);
#define fst first
#define snd second
#define pb push_back
#define mp make_pair
typedef long long ll;
typedef pair<int,int> pii;

#define INF 987654321
#define MAX 101

int main() {
    int n, k, p;
    cin >> n >> k >> p;
    int cnt, sol = 0;
    for (int i=0; i<n; i++) {
        cnt = 0;
        for (int j=0; j<k; j++) {
            int a;
            cin >> a;
            if (a == 0) cnt++;
        }
        if (cnt < p) sol++;
    }
    cout << sol;
}